hw/ip/pattgen/doc/dv/index.md

```diff
@@ -68,14 +76,22 @@ The `pattgen_base_vseq` virtual sequence is extended from `cip_base_vseq` and se
 All test sequences are extended from `pattgen_base_vseq`.
 It provides commonly used handles, variables, functions and tasks that the test sequences can simple use / call.
 Some of the most commonly used tasks / functions are as follows:
-* task 1:
-* task 2:
+* task 1: setup_pattgen_channel
+* task 2: start_pattgen_channels
+* task 3: stop_pattgen_channels
+* task 4: control_channels
```

Comment:
Could you write a short (one sentence) description of each task? Maybe you can also remove the `task *`, you usually just do this:
```
* setup_pattgen_channel: description 1
```